### ISSUE TITLE
SELECT output rendered incorrectly?

### ISSUE THREAD

**Issue by cmeeren** [DbUp/DbUp#606]
Having trouble with a script, so I inserted the following, hoping to get some output that could shed light on why it fails:

```sql
SELECT * 
  FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS 
 WHERE TABLE_NAME = 'MyTable'
```

This caused the following output:

```
[14:01:49 INF] ------------------------------------------------------------------------------------------------------------------------------------
[14:01:49 INF] | {0, 6} | {1, 9} | {2, 33} | {3, 6} | {4, 9} | {5, 20} | {6, 13} | {7, 4} | {8, 4} |
[14:01:49 INF] ------------------------------------------------------------------------------------------------------------------------------------
```

Not sure exactly what's being shown here, but it certainly isn't the actual selected data.

I am using `.LogScriptOutput()` and dbup-sqlserver 4.5.0.

**Comment by sungam3r** [comment]
Fixed by #543 ?

**Comment by cmeeren** [comment]
Seems like it. Thanks! :)
